readme.md

```diff
@@ -1,16 +1,84 @@
 # Comment Notifications
 
-[![Build Status](https://secure.travis-ci.org/silverstripe/comment-notifications.png?branch=master)](http://travis-ci.org/silverstripe/comment-notifications)
+[![Build Status](https://travis-ci.org/silverstripe/comment-notifications.svg?branch=master)](https://travis-ci.org/silverstripe/comment-notifications)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/silverstripe/comment-notifications/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/silverstripe/comment-notifications/?branch=master)
+[![Code Coverage](https://scrutinizer-ci.com/g/silverstripe/comment-notifications/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/silverstripe/comment-notifications/?branch=master)
+[![Build Status](https://scrutinizer-ci.com/g/silverstripe/comment-notifications/badges/build.png?b=master)](https://scrutinizer-ci.com/g/silverstripe/comment-notifications/build-status/master)
+[![codecov.io](https://codecov.io/github/silverstripe/comment-notifications/coverage.svg?branch=master)](https://codecov.io/github/silverstripe/comment-notifications?branch=master)
 
-Provides simple comment notifications
+[![Latest Stable Version](https://poser.pugx.org/silverstripe/comment-notifications/version)](https://packagist.org/packages/silverstripe/comment-notifications)
+[![Latest Unstable Version](https://poser.pugx.org/silverstripe/comment-notifications/v/unstable)](//packagist.org/packages/silverstripe/comment-notifications)
+[![Total Downloads](https://poser.pugx.org/silverstripe/comment-notifications/downloads)](https://packagist.org/packages/silverstripe/comment-notifications)
+[![License](https://poser.pugx.org/silverstripe/comment-notifications/license)](https://packagist.org/packages/silverstripe/comment-notifications)
+[![Monthly Downloads](https://poser.pugx.org/silverstripe/comment-notifications/d/monthly)](https://packagist.org/packages/silverstripe/comment-notifications)
+[![Daily Downloads](https://poser.pugx.org/silverstripe/comment-notifications/d/daily)](https://packagist.org/packages/silverstripe/comment-notifications)
+
+[![Dependency Status](https://www.versioneye.com/php/silverstripe:comment-notifications/badge.svg)](https://www.versioneye.com/php/silverstripe:comment-notifications)
+[![Reference Status](https://www.versioneye.com/php/silverstripe:comment-notifications/reference_badge.svg?style=flat)](https://www.versioneye.com/php/silverstripe:comment-notifications/references)
+
+![codecov.io](https://codecov.io/github/silverstripe/comment-notifications/branch.svg?branch=master)
```

Comment:
Can we just have Travis, Scrutinizer and Codecov.io please? If we want to standardise past that we should discuss, but the comments module (this) is super excessive

Reply:
Just figured keeping it consistent with comments (copied from that) but will remove